runtime/lua/vim/filetype.lua

```diff
@@ -774,6 +774,12 @@ local extension = {
   xml = function() vim.fn["dist#ft#FTxml"]() end,
   y = function() vim.fn["dist#ft#FTy"]() end,
   zsql = function() vim.fn["dist#ft#SQL"]() end,
+  --helpfiles match *.txt, but should have a modeline as last line
+  txt = function(path, bufnr)
```

Comment:
Move this comment inside the function

```suggestion
  txt = function(path, bufnr)
    -- helpfiles match *.txt, but should have a modeline as last line
```